assets/prototype/domain/eventEditor/datetimes/datesList/DatesList.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { H3 } from '@blueprintjs/core/lib/esm';
+import DatesListFilterBar from './filterBar/DatesListFilterBar';
+import AddNewDateButton from './AddNewDateButton';
+import DateCard from '../dateCard/DateCard';
+
+/**
+ * Internal dependencies
+ */
+import { Datetime } from '../../data/types';
+
+const listStyle = {
+	display: 'flex',
+	flexFlow: 'row wrap',
+	justifyContent: 'space-between',
+	width: '100%',
+};
+
+interface DatesListProps {
+	datetimes: Datetime[];
+}
+
+const DatesList = ({ datetimes }: DatesListProps) => {
```

Comment:
Also, as discussed earlier, we should prefer `DatetimeList` instead of just `DatesList`